set.mm

```diff
@@ -437501,6 +437562,9 @@ orthogonal vectors (i.e. whose inner product is 0) is the sum of the
  althtmldef "UPWalks" as "UPWalks";
  latexdef "UPWalks" as "\mathrm{UPWalks}";
 /* Set of unordered pairs */
+htmldef "<>" as '<>';
```

Comment:
This can be mixed up with some kind of "not equal".  Have a look here http://xahlee.info/comp/unicode_arrows.html for a replacement.  My favorite would be U+21CB ( ⇋ ).

Reply:
(note that the unicode display uses althtmldef which says `&#8644;` ⇄ instead of `<>` <>)

Reply:
ok then.